docs/SME.md

```diff
@@ -207,6 +207,7 @@ Pros:
 Cons:
 - .NET developer wrongly puts or forgets to put the streaming attribute on a method. We have already seen above that this should not affect the correctness characteristics, but can affect performance of the methods.
 - Although we called out that static analyzers can be used to identify errors made by developers by passing VL-dependent objects between different streaming state methods, lot of projects might turn off static analyzers and for them, the behavior in such scenario is undefined. We could still utilize the method attribute information to analyze if VL-dependent arguments are passed or returned and add `throw InvalidProgramException` or something similar.
+- It will not be possible to call exisiting library routines whilst in streaming mode, unless those routines are explicitly marked as `SME_STREAMING_COMPATIBLE`. This could cause a cascade as users raise issues to request more and more routines to be marked as streaming compatible.
```

Comment:
hhm, i think calling existing routines from "streaming" methods should just work, because for them callee is non-streaming method and the codegen will insert `SMSTOP` before such call and `SMSTART` after the call. Isn't it?